docs/helpers.rst

```diff
@@ -462,11 +463,12 @@ If you use type annotations, you can annotate the fixture like this::
 ``django_assert_max_num_queries``
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. py:function:: django_assert_max_num_queries(num, connection=None, info=None)
+.. py:function:: django_assert_max_num_queries(num, connection=None, info=None, using=None)
 
   :param num: expected maximum number of queries
   :param connection: optional non-default DB connection
   :param str info: optional info message to display on failure
+  :param str using: optional the non-default DB connection name
```

Comment:
```suggestion
  :param connection: optional database connection
  :param str info: optional info message to display on failure
  :param str using: optional database connection name
```

Reply:
Ditto.

I’d change this if the maintainer wants it.